src/frontend/src/lib/utils/format.utils.ts

```diff
@@ -68,6 +68,30 @@ export const formatNanosecondsToDate = (nanoseconds: bigint): string => {
 	return date.toLocaleDateString('en', DATE_TIME_FORMAT_OPTIONS);
 };
 
+export const formatSecondsToNormalizedDate = ({
+	seconds,
+	currentDate
+}: {
+	seconds: number;
+	currentDate?: Date;
+}): string => {
+	const date = new Date(seconds * 1000);
+	const today = currentDate ?? new Date();
+	const daysDifference = Math.ceil((date.getTime() - today.getTime()) / (1000 * 60 * 60 * 24));
```

Comment:
Not sure if it would becomes too verbose but we have some constants like `SECONDS_IN_MINUTE` etc.

Reply:
makes sense, done